sessionctx/variable/session.go

```diff
@@ -446,6 +446,8 @@ type SessionVars struct {
 	// LockWaitTimeout is the duration waiting for pessimistic lock in milliseconds
 	// negative value means nowait, 0 means default behavior, others means actual wait time
 	LockWaitTimeout int64
+
+	NeedntCBOPointGet bool
```

Comment:
Why we need this varible?

Reply:
Oh, that's the code of debug.

Reply:
Sorry, the variable is to ban CBO point get for `insert/delete/update`.